src/documents/sanity_checker.py

```diff
@@ -9,125 +11,135 @@
-                        f"Checksum mismatch of document {doc.pk}. "
+                        doc.pk,
+                        "Checksum mismatch. "
                         f"Stored: {doc.checksum}, actual: {checksum}.",
                     )
 
         # Check sanity of the archive file.
-        if doc.archive_checksum and not doc.archive_filename:
+        if doc.archive_checksum is not None and doc.archive_filename is None:
             messages.error(
-                f"Document {doc.pk} has an archive file checksum, but no "
-                f"archive filename.",
+                doc.pk,
+                "Document has an archive file checksum, but no archive filename.",
             )
-        elif not doc.archive_checksum and doc.archive_filename:
+        elif doc.archive_checksum is None and doc.archive_filename is not None:
             messages.error(
-                f"Document {doc.pk} has an archive file, but its checksum is "
-                f"missing.",
+                doc.pk,
+                "Document has an archive file, but its checksum is missing.",
             )
         elif doc.has_archive_version:
-            if not os.path.isfile(doc.archive_path):
-                messages.error(f"Archived version of document {doc.pk} does not exist.")
+            archive_path: Final[Path] = Path(doc.archive_path).resolve()
+            if not archive_path.exists() or not archive_path.is_file():
+                messages.error(doc.pk, "Archived version of document does not exist.")
             else:
-                if os.path.normpath(doc.archive_path) in present_files:
-                    present_files.remove(os.path.normpath(doc.archive_path))
+                if archive_path in present_files:
+                    present_files.remove(archive_path)
                 try:
-                    with doc.archive_file as f:
-                        checksum = hashlib.md5(f.read()).hexdigest()
+                    checksum = hashlib.md5(archive_path.read_bytes()).hexdigest()
                 except OSError as e:
                     messages.error(
-                        f"Cannot read archive file of document {doc.pk}: {e}",
+                        doc.pk,
+                        f"Cannot read archive file of document : {e}",
                     )
                 else:
                     if not checksum == doc.archive_checksum:
                         messages.error(
-                            f"Checksum mismatch of archived document "
-                            f"{doc.pk}. "
+                            doc.pk,
+                            "Checksum mismatch of archived document. "
                             f"Stored: {doc.archive_checksum}, "
                             f"actual: {checksum}.",
                         )
 
         # other document checks
         if not doc.content:
-            messages.info(f"Document {doc.pk} has no content.")
+            messages.info(doc.pk, "Document has no content.")
```

Comment:
```suggestion
            messages.info(doc.pk, "Document contains no OCR data.")
```

Reply:
Weird, I added it to the batch, but it didn't commit I guess